ads/google/a4a/utils.js

```diff
@@ -54,9 +54,10 @@ const AmpAdImplementation = {
 
 /** @const {!Object} */
 export const ValidAdContainerTypes = {
-  'AMP-STICKY-AD': 'sa',
+  'AMP-CAROUSEL': 'ac',
```

Comment:
At this point, they're kind of entangled, and the changes are isolated such that it should (hopefully) not be too burdensome on the reviewer. If you disagree strongly, I can spend some time splitting those off into a separate PR, though.